src/logging.cc

```diff
@@ -161,6 +161,8 @@ static const char* DefaultLogDir() {
   return "";
 }
 
+GLOG_DEFINE_int32(logfile_mode, 0644, "Log file mode/permissions");
```

Comment:
Haha! I'm just messing around :dancer: ...but the gflags default help _could_ use some love.